packages/manager/apps/web/package.json

```
@@ -27,6 +27,7 @@
     "@ovh-ux/manager-exchange": "^1.0.0",
     "@ovh-ux/manager-navbar": "^2.1.3",
     "@ovh-ux/manager-office": "^1.0.0",
+    "@ovh-ux/manager-product-offers": "^0.0.0",
```

```suggestion
    "@ovh-ux/manager-product-offers": "^0.0.0 || ^1.0.0",
```
or 
```suggestion
    "@ovh-ux/manager-product-offers": "^0.0.0 || ^0.1.0",
```
if your component is released